beets/ui/__init__.py

```diff
@@ -773,6 +774,26 @@ def add_all_common_options(self):
 # succinct example program.
 
 
+# FIXME: kill original Subcommand in favor of this
```

Comment:
Good question. It would be nice if it's easy—and yes, I would love to look into writing that shim for compatibility—but not essential (as long as we port all the included plugins, of course).